app/export/csv.js

```diff
@@ -1,9 +1,11 @@
 const { getMicrochip, extractLatestAddress, extractLatestInsurance, extractEmail, extractLatestPrimaryTelephoneNumber, extractLatestSecondaryTelephoneNumber, truncDate } = require('../dto/dto-helper')
 
-const convertToCsv = (rows) => {
+const convertToCsv = (rows, removeHeader = false) => {
   const csvRows = []
 
-  csvRows.push(headerRow)
+  if (!removeHeader) {
+    csvRows.push(headerRow)
```

Comment:
Know headerRow is probably hoisted, but would be more readable if headerRow was created before this.